data/formats-data.js

```diff
@@ -6108,7 +6108,7 @@ exports.BattleFormatsData = {
 		tier: "New",
 	},
 	tapulele: {
-		randomBattleMoves: ["moonblast", "psyshock", "calmmind", "focusblast", "aromatherapy"],
+		randomBattleMoves: ["moonblast", "psyshock", "calmmind", "focusblast", "taunt", "psychic"],
```

Comment:
Psychic is still really popular on the Lele, but then again, most people run choiced. And there isn't enough useful moves to build a full choiced set for Lele (most put HP Fire for Scizor as the last move, but it isn't that great outside a choice set)